exercises/hello-world/example.go

```diff
@@ -2,7 +2,8 @@ package hello
 
 const testVersion = 2
 
-func HelloWorld(name string) string {
+// World says hello to given name or world.
+func World(name string) string {
```

Comment:
Now that the comment exists, I'm going to get all picky about it :)

How about:

> World says hello to the given name.
> If no name is passed, it greets the world.